queue_job/job.py

```diff
@@ -660,21 +687,28 @@ def set_enqueued(self):
         self.date_enqueued = datetime.now()
         self.date_started = None
         self.worker_pid = None
+        self.worker_hostname = None
+        self.db_backend_pid = None
 
     def set_started(self):
         self.state = STARTED
         self.date_started = datetime.now()
         self.worker_pid = os.getpid()
+        self.worker_hostname = gethostname()
+        self.db_backend_pid = self.get_db_backend_pid()
 
     def set_done(self, result=None):
         self.state = DONE
         self.exc_info = None
         self.date_done = datetime.now()
+        self.worker_pid = None
+        self.db_backend_pid = None
```

Comment:
Should we align resetting the data to how it's done for `worker_pid`? That would mean not resetting them here and in `set_failed`. I think that makes sense as the data can be used for log filtering and debugging.

Reply:
Agreed. I though it was useless afterwards, but I see your point!